### PR TITLE
Shiviagarw large scale

### PR DESCRIPTION
**Pull request by Shiviagarw**
Adding summary

### CODE REVIEW

File: _summaries/liu2019large-scale_2.md [hunk 1]
```diff
@@ -1,19 +1,90 @@
...
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+## The Core Idea of this paper?
+Real world data is long tailed and open-ended distributed. A practical recognition system must classify among majority and minority classes. 
+This paper discusses about the Open Long-Tailed Recognition (OLTR) as learning from naturally distributed data and optimizing the classification 
+accuracy over a balanced test set including head, tail, and open classes. OLTR focuses on imbalanced classification, few-shot learning, and open-set
```
Comment (mattkelleher) [inline _summaries/liu2019large-scale_2.md:13]: Might be good to define what head, tail, and open classes are for people who may be less familiar with the terminology. There is a nice figure (figure 1) in the paper that could be paired with a sentence or two to nicely describe these 3 types of classes.

File: _summaries/liu2019large-scale_2.md [hunk 2]
```diff
@@ -1,19 +1,90 @@
...
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+## The Core Idea of this paper?
+Real world data is long tailed and open-ended distributed. A practical recognition system must classify among majority and minority classes. 
+This paper discusses about the Open Long-Tailed Recognition (OLTR) as learning from naturally distributed data and optimizing the classification 
+accuracy over a balanced test set including head, tail, and open classes. OLTR focuses on imbalanced classification, few-shot learning, and open-set 
+recognition in single integrated algorithm, however the existing classification approaches focus only on one aspect and deliver poorly. The key 
+challenges for OLTR are tail recognition robustness and open-set sensitivity. The core idea of this paper is to develop an integrated OLTR algorithm
```
Comment (mattkelleher) [inline _summaries/liu2019large-scale_2.md:15]: Not something you necessarily have to add to this summary but might be good to think about for the discussion: This paper trys to tackle a large problem in 1 shot. What is the motivation for this as opposed to using multiple models that are optimized for their own tasks (head, tail, out).

File: _summaries/liu2019large-scale_2.md [hunk 3]
```diff
@@ -1,19 +1,90 @@
...
+that maps an image to a feature space such that visual concepts. It can relate to each other based on a learned metric that respects the closed-world 
+classification while acknowledging the novelty of the open world. This dynamic meta-embedding combines a direct image feature and an associated memory 
+feature, with the feature norm indicating the familiarity to known classes. It has modulated attention to encourage the head and tail classes to use 
+different sets of spatial features. As the Meta embedding relates head and tail classes, the modulated attention maintains discrimination between them.
+
+
+## Model and Technical representation:
+
+The OLTR consists of two main modules, dynamic Meta embedding and modulated attention. The former relates and transfers knowledge between head and tail 
+classes and the latter maintains discrimination between them.
```
Comment (mattkelleher) [inline _summaries/liu2019large-scale_2.md:25]: what about the open tail? (never before seen objects). Could be a good idea to briefly mention how these rare cases are handled

File: _summaries/liu2019large-scale_2.md [hunk 4]
```diff
@@ -1,19 +1,90 @@
...
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+•  The  OLTR task  learns from natural long-tail open-end distributed data and optimizes the overall accuracy over a balanced test set.
+
+•  The integrated OLTR algorithm can be used, with dynamic meta-embedding, in order to share visual knowledge between head and tail classes and to reduce 
+   confusion between tail and open classes
+
+•  It gives good accuracy over ImageNet-LT.
```
Comment (mattkelleher) [inline _summaries/liu2019large-scale_2.md:90]: maybe could combine points 2 and 3 and add an additional point pointing out how this paper is the first that sets up datasets for OLTR problem and can serve as a benchmark for future works in OLTR.

File: _summaries/liu2019large-scale_2.md (hunk 1 above)
Comment (Shiviagarw) [reply _summaries/liu2019large-scale_2.md:13]: Added as per your comment.

File: _summaries/liu2019large-scale_2.md (hunk 2 above)
Comment (Shiviagarw) [reply _summaries/liu2019large-scale_2.md:15]: Thankyou for the review. I believe adding these points can give clearer picture of the core idea of this paper. I will definitely agree with you at the time of discussion.

File: _summaries/liu2019large-scale_2.md (hunk 3 above)
Comment (Shiviagarw) [reply _summaries/liu2019large-scale_2.md:28]: I have mentioned all the details of implementation I could find in this paper. Couldn't find anything specific to open tail. The rare cases are handled by . Dynamic Meta-Embedding and attention . I have already briefly described these points.

File: _summaries/liu2019large-scale_2.md (hunk 4 above)
Comment (Shiviagarw) [reply _summaries/liu2019large-scale_2.md]: Done the change as suggest by you.

**Comment by zhaoyue-zephyrus** [comment]
@nilesh2797  any comments on this? (though summary is submitted late)

**Review by zhaoyue-zephyrus — APPROVED** [review]
